<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="mainwidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="messagewidget" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>541</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="MessagetextBrowser">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>431</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1215</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>通信设置</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>通信协议</string>
    </property>
    <addaction name="actionUDS"/>
    <addaction name="actionCCP"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>参数设置</string>
   </property>
   <property name="iconText">
    <string>参数设置</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>断开连接</string>
   </property>
  </action>
  <action name="actionUDS">
   <property name="text">
    <string>UDS</string>
   </property>
  </action>
  <action name="actionCCP">
   <property name="text">
    <string>CCP</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>显示原始数据</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>关闭原始数据</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>硬件测试</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
